Scenes/HUD/BottomMenuBar.tscn

```diff
@@ -236,17 +131,8 @@ grow_vertical = 2
 mouse_force_pass_scroll_events = false
 theme = ExtResource("1_r4ocg")
 script = ExtResource("6")
-_item_stash_menu = NodePath("HBoxContainer/CentralMenu2/VBoxContainer/MarginContainer/ScrollContainer/StashMarginContainer/ItemStashMenu")
```

Comment:
Removal of all of these export vars is incorrect because they are still used in script.
Causes null errors during runtime.
For example, if you press on the "Show only regular items" button you get this error:

> Invalid call. Nonexistent function 'set_current_item_type_filter' in base 'Nil'.

Reply:
I will remove them completely in the next commits.